code/datums/diseases/advance/advance.dm

```diff
@@ -137,6 +137,15 @@
 	infect(infectee, make_copy)
 	return TRUE
 
+/datum/disease/advance/after_add()
+	if(affected_mob)
+		RegisterSignal(affected_mob, COMSIG_LIVING_DEATH, .proc/on_mob_death) //RegisterSignal(target, COMSIG_LIVING_DEATH, PROC_REF(on_death))
```

Comment:
this needs to be converted to PROC_REF or it won't run on the server

Reply:
worked local, im gonna take your word for it and do the changes, tomorrow.